src/player.py

```diff
@@ -3,23 +3,34 @@
 class Player(Game):
     # initialize hand with games deck
     def __init__(self, game):
+        print("I compile")
 
     # lay down, should just call players hand easily
+    # put down a full set, or add to a set that is already laid down
     def lay_down(self):
+        print("I compile")
```

Comment:
this method and the laydown method in game have too similar names. I think we should consider renaming the one in game to display special method